R/load_pop_data.R

```diff
@@ -107,54 +104,51 @@ set_age <- function(object, ...){
 }
 
 #' @export
```

Comment:
if the `set_*` methods are going to be exported, they need to be documented. However, I don't think they need to be exported?